README.md

```diff
@@ -129,10 +129,15 @@ In the example above, the response time baseline defines `http_time_daily` as ou
 As result, the service will generate two measurements: `http_time_daily_15d` and `http_time_daily_30d`.
 The measurements contain two fields: `value`, which is the baseline and `stddev` which is the standard deviation.
 
-By default the baseline service will preserve all tags from the input measurement.
+By default, the baseline service will preserve all tags from the input measurement.
 When this is not the intended behaviour, it is possible to keep only certain tags (or none).
 The values of all other tags will be aggregated together.
 
 For example, if we assume that the `http_requests_count` measurement has two tags (`http_path` and `http_status`),
 we can specify `tags: [http_path]` as shown above. This means that the baseline will be generated for each http_path individually,
 however the `http_status` will not be used for differentiation.
+
+## SBOM
+
+To generate a software bill of materials (SBOM), execute the gradle task cyclonedxBom.
```

Comment:
Please wrap the task name in `` ` ``  characters.

Example:
...the gradle task `cyclonedxBom`.